src/app/components/Loader/styles.module.scss

```diff
@@ -0,0 +1,19 @@
+@import 'variables/colors';
+
+.loader-container {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100vw;
+  height: 100vh;
+  z-index: 10000;
+  opacity: 0.6;
```

Comment:
Add to variables scss file, maybe you need this same opacity in another component